src/apollo/client.ts

```diff
@@ -0,0 +1,10 @@
+import { ApolloClient } from "apollo-client";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { HttpLink } from "apollo-link-http";
+
+export const client = new ApolloClient({
+  link: new HttpLink({
+    uri: "https://api.thegraph.com/subgraphs/name/tomprsn/yearn-vaults-v2-subgraph-mainnet"
+  }),
+  cache: new InMemoryCache()
+});
```

Comment:
Could we move this file into `services`?

Reply:
done